File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int sum=0;
        vector<int>prefix;
        for(int i=0;i<nums.size(); i++){
            sum+=nums[i];
            prefix.push_back(sum);
        }return prefix;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        n=len(nums)
        ans=[]
        ans.append(nums[0])
        for i in range(1,n):
            x=nums[i]+ans[i-1]
            ans.append(x)
        return ans
        